interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,506 @@ components:
+            required. OVNs for constraints are required if and only if the USS
+            managing this operational intent is performing the constraint
+            processing role, which is indicated by whether the subscription
+            associated with this operational intent triggers notifications for
+            constraints.  The key does not need to contain the OVN for the
+            operational intent being updated.
+          anyOf:
+          - $ref: '#/components/schemas/Key'
+        state:
+          $ref: '#/components/schemas/OperationalIntentState'
+        uss_base_url:
+          $ref: '#/components/schemas/OperationalIntentUssBaseURL'
+
+        subscription_id:
+          description: >-
+            The ID of an existing subscription that the USS will use to keep the
+            operator informed about updates to relevant airspace information. If
+            this field is not provided when the operational intent is in the
+            Activated, Nonconforming, or Contingent state, then the
+            `new_subscription` field must be provided in order to provide
+            notification capability for the operational intent.  The
+            subscription specified by this ID must cover at least the area over
+            which this operational intent is conducted, and it must provide
+            notifications for operational intents.
+          anyOf:
+          - $ref: '#/components/schemas/EntityID'
+        new_subscription:
+          description: >-
+            If an existing subscription is not specified in `subscription_id`,
+            and the operational intent is in the Activated, Nonconforming, or
+            Contingent state, then this field must be populated.  When this
+            field is populated, an implicit subscription will be created and
+            associated with this operational intent, and will generally be
+            deleted automatically upon the deletion of this operational intent.
+          anyOf:
+          - $ref: '#/components/schemas/ImplicitSubscriptionParameters'
+
+    FlightAuthorisationData:
+      description: |-
+        A dataset to hold details of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:   
+          $ref: '#/components/schemas/OperationMode'
+        operation_category:
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            enum:
+              - network
```

Comment:
Best practice for most enums is to have "Undeclared" or "Unknown" as the first option because some code generation will translate enums to integers and unspecified values become 0.  Also, to align with ASTM API style, let's capitalize the first letter of each value.

Reply:
Thank you, I have now updated to add `undeclared` and capitalized the other two.